instructor_notes/openscpca/create-an-analysis-module.md

```diff
@@ -0,0 +1,38 @@
+
+# Instructor Notes: Making an analysis module Activity
+
+
+## Learning goals
+
+- Gain more experience creating and working in a feature branch
+- Gain experience creating a module with a conda environment
+- Gain familiarity with module documentation
+
+## Activity
+
+For this activity, instructors will demonstrate creating a module while participants actively follow along.
+The lead instructor should, like particpants, work in their fork of `OpenScPCA-analysis` and lead participants through these steps:
+
+1. Ensure your fork's `main` branch is up-to-date, following [these instructions](https://openscpca.readthedocs.io/en/latest/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream/).
+
+2. Have a look at the issue in `AlexsLemonade/OpenScPCA-analysis` created for the workshop
+    - Discuss what information we learn from the issue
+    - Note that the issue _is not_ about completing an entire analysis top to bottom, but focuses on a highly modular, if you will, task: Establishing a module
+
+3. Locally, check out a feature branch named roughly `<username>/<issue-number>-create-<module-name>`
```

Comment:
We do this out of habit, but since everyone is working in their own branch, maybe `<issue-number>-create-<module-name>` is sufficient? i.e. I don't think people necessarily need their username prefix if they are working in their own forks.

Reply:
If it is the docs this way then we should match it.